

class PyCUIOutOfBoundsError(Exception):
    pass


class PyCUIError(Exception):
    pass